ruby-training/trainees/alan/test_katas/05/Sum_with_out_max_min/sum_with_out_max_min.rb

```diff
@@ -0,0 +1,15 @@
+
+class SumWithOutMaxMin
+    
+   
+    def self.remove_min_and_max(array_of_intergers)
+        sorted_array = array_of_integers.sort
+        sorted_array.shift
```

Comment:
Ideally, we avoid using methods that directly mutate an existing variable/object/data structure. First you called it `sorted_array` but by the time you returned it you executed 2 other operations on it, making the initial variable name less relevant and possibly confusing anyone who comes in contact with this code.
```suggestion
        shifted_array = sorted_array.shift
```

Reply:
If I assign the array in that way, the variable will  be the poped or shifted value not the array.